packages/arb-token-bridge-ui/src/components/TransactionHistory/TransactionsTableRow.tsx

```diff
@@ -0,0 +1,189 @@
+  CheckCircleIcon,
+  XCircleIcon
+} from '@heroicons/react/24/outline'
+
+import { DepositStatus, MergedTransaction } from '../../state/app/state'
+import { formatAmount } from '../../util/NumberUtils'
+import { sanitizeTokenSymbol } from '../../util/TokenUtils'
+import { getWagmiChain } from '../../util/wagmi/getWagmiChain'
+import {
+  ChainId,
+  getExplorerUrl,
+  getNetworkName,
+  isNetwork
+} from '../../util/networks'
+import { NetworkImage } from '../common/NetworkImage'
+import { isTxClaimable, isTxFailed, isTxPending } from './helpers'
+import { ExternalLink } from '../common/ExternalLink'
+import { Button } from '../common/Button'
+import { TransactionsTableRowAction } from './TransactionsTableRowAction'
+import { AssetType } from '../../hooks/arbTokenBridge.types'
+
+export function TransactionsTableRow({ tx }: { tx: MergedTransaction }) {
+  const sourceChainId = tx.cctpData?.sourceChainId ?? ChainId.ArbitrumOne
+
+  const { isEthereumMainnetOrTestnet: isSourceChainIdEthereum } =
+    isNetwork(sourceChainId)
+
+  const isClaimableTx = tx.isCctp || tx.isWithdrawal
+
+  const txRelativeTime = dayjs(tx.createdAt).fromNow()
+
+  const tokenSymbol = useMemo(
+    () =>
+      sanitizeTokenSymbol(tx.asset, {
+        erc20L1Address: tx.tokenAddress,
+        chain: getWagmiChain(
+          isSourceChainIdEthereum ? tx.parentChainId : tx.childChainId
+        )
+      }),
+    [
+      tx.asset,
+      tx.tokenAddress,
+      tx.parentChainId,
+      tx.childChainId,
+      isSourceChainIdEthereum
+    ]
+  )
+
+  const StatusLabel = useCallback(() => {
+    if (isTxFailed(tx)) {
+      return (
+        <div className="flex items-center space-x-1 text-red-400">
+          <XCircleIcon height={14} className="mr-1" />
+          <span>Failed</span>
+          <ExternalLink
+            href={`${getExplorerUrl(tx.parentChainId)}/tx/${tx.txId}`}
+            className="mb-1"
+          >
+            <ArrowTopRightOnSquareIcon height={10} />
+          </ExternalLink>
+        </div>
+      )
+    }
+
+    if (isTxPending(tx)) {
+      const sourceChainId = tx.isWithdrawal ? tx.childChainId : tx.parentChainId
+
+      return (
+        <div className="flex items-center space-x-1 text-yellow-400">
+          <div className="mr-1 h-[10px] w-[10px] rounded-full border border-yellow-400" />
+          <span>Pending</span>
+          <ExternalLink
+            href={`${getExplorerUrl(sourceChainId)}/tx/${tx.txId}`}
+            className="mb-1"
+          >
+            <ArrowTopRightOnSquareIcon height={10} />
+          </ExternalLink>
+        </div>
+      )
+    }
+
+    if (isTxClaimable(tx)) {
+      return (
+        <div className="flex items-center space-x-1 text-green-400">
+          <div className="mr-1 h-[10px] w-[10px] rounded-full border border-green-400" />
+          <span>Claimable</span>
+          <ExternalLink
+            href={`${getExplorerUrl(tx.parentChainId)}/tx/${tx.txId}`}
+            className="mb-1"
```

Comment:
on figma there's actually no margin at the bottom and the centre of the icon image on Figma was aligned to the text at centre, so perhaps we should double check with designer that this alignment is intended?

it actually looks out of place to not be centered

Reply:
I centered it, we can ask the designer later.